packages/@blockprotocol/core/src/types.ts

```diff
@@ -217,3 +217,8 @@ export type ServiceDefinition = {
   coreVersion: string;
   messages: ServiceMessageDefinition[];
 };
+
+export type HtmlBlockDefinition = {
+  url: string;
+  source?: string;
+};
```

Comment:
I think it could be made more obvious what these keys are for and what's going to happen with them, which could be a combination of (a) renaming them, (b) docstring on this type, (c) docstring on `renderHtmlBlock`, e.g.:
1. Specify that the `url` is the url of the block's entry point source file
2. Specify that `source` is a sort of 'source override', whereby this source will be used in lieu of whatever is actually at `url`
3. (This is more for `renderHtmlBlock`): specify what actually goes on when different combination of variables are provided, e.g. base URL will be inferred from `url` and used to rewrite paths (in all cases), and source used depends on presence of `source` or not

Reply:
👍